WordPress/Classes/ViewRelated/Stats/Shared Views/StatsTotalRow.swift

```diff
@@ -228,9 +228,9 @@ class StatsTotalRow: UIView, NibLoadable, Accessible {
 
         let itemTitle = itemLabel.text ?? ""
         let dataTitle = dataLabel.text?.accessibilityLabel ?? dataLabel.text ?? ""
-        let secondDataTitle = secondDataLabel.text?.accessibilityLabel ?? secondDataLabel.text ?? ""
+        let secondDataTitle = secondDataLabel.text?.accessibilityLabel ?? secondDataLabel.text
 
-        accessibilityLabel = [itemTitle, dataTitle, secondDataTitle].joined(separator: ", ")
+        accessibilityLabel = [itemTitle, dataTitle, secondDataTitle].compactMap { $0 }.joined(separator: ", ")
```

Comment:
nitpick: VoiceOver reads each row as "My blog post, 25, 1" which is fine, but I wonder if users would expect "My blog post, 25 opens and 1 click". The complexity comes from this `StatsTotalRowData` being used in so many other places and also the handling of plurals and localization. This might be best considered at a broader level outside the scope of this small project.

Reply:
Yes. Even passing a localized string from ViewModel into the row wouldn't help since as you mentioned we need to deal with plurals as well. The next best thing coud be to at least emphasize column names.  "My blog post. Opens: 25. Clicks: 1."